src/components/AboutTeam/useStyles.js

```diff
@@ -0,0 +1,77 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {
+    backgroundColor: "#f8f8f8",
+    padding: `${typography.pxToRem(40)} 0`,
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(80)} 0`,
+    },
+  },
+  cardRoot: {
+    boxShadow: "none",
+    backgroundColor: "unset",
+    maxWidth: typography.pxToRem(350),
+    padding: `0 ${typography.pxToRem(20)}`,
+    [breakpoints.up("md")]: {
+      maxWidth: typography.pxToRem(296),
+      marginRight: typography.pxToRem(16),
+      padding: 0,
+    },
+    [breakpoints.up("lg")]: {
+      maxWidth: typography.pxToRem(376),
```

Comment:
Where do these maxWidths come from and why are we using them? For example, in mobile, don't we want the card to be based on screen size?

Reply:
Fixed